apps/snitch_core/test/domain/order/transitions_test.exs

```diff
@@ -141,4 +142,20 @@ defmodule Snitch.Domain.Order.TransitionsTest do
       assert {:ok, %{packages: []}} = Repo.transaction(result.multi)
     end
   end
+
+  describe "associate_package" do
+    setup do
+      [order: insert(:order, user: build(:user))]
+    end
+
+    test "when packages are empty", %{order: order} do
+      result =
```

Comment:
You will have to call `Mox.expect` here. Line 155 will eventually result in a call to `MoneyTools.zero`, which calls the `Tools.Defaults.fetch`, to fetch the default currency.
Refer the `Snitch.Tools.MoneyTest` module.